### DISCUSSION TITLE
Can webots send request to control the Logitech G29 steering wheel?

### DISCUSSION THREAD

**Discussion by tian96-pu** [cyberbotics/webots#6431] [General]
I am designing a driving simulator using webots, and Logitech G29. I would like to design automated driving while human does not need to put their hands all the time on the steering wheel. Meanwhille, the vehicle in the webots can be driven by itself, which controls the steering wheel and acceleratation automatically. 
My question is, can webots sent out request to control the steering wheel angle of Logitech G29 physically?

Thanks.

**Comment by ShuffleWire** [comment]
Webots can't by itself, but if you have the driver/API to do it (like a function as `logitech_g29_turn_steering_wheel()` ) then you can very well write a controller that use it. When the simulation says that the steering will should turn, then call whatever function of your API to make your hardware move.
